deployment/scripts/install.sh

```diff
@@ -303,12 +303,12 @@ if [ "${ENABLE_OPS_CLUSTER}" == true ]; then
 	oc process logging-curator-ops-template | oc create -f -
 fi
 
-echo "(Re-)Creating deployed objects"
+#echo "(Re-)Creating deployed objects"
 # this is separate from the support-pre-template objects as we aren't automatically recreating the route and oauthclient
-if [ "${KEEP_SUPPORT}" != true ]; then
-	oc delete oauthclient,route --selector logging-infra=support
-	oc process logging-support-template | oc create -f -
-fi
+#if [ "${KEEP_SUPPORT}" != true ]; then
+#	oc delete oauthclient,route --selector logging-infra=support
+#	oc process logging-support-template | oc create -f -
+#fi
```

Comment:
yeah might as well nuke all this... i think it might have gotten in via rebase merge conflicts.